webpack/loaders.js

```diff
@@ -4,7 +4,7 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const JSLoader = {
   test: /^(?!.*\.component\.js$)(?!.*\.stories\.js$).*\.js$/,
   include: [
-    path.resolve(__dirname, "components")
+    path.join(__dirname, '../', 'components')
```

Comment:
you'll want to use https://nodejs.org/api/path.html#path_path_sep here instead of `../`. Use `path.sep` so that windows works.